packages/m2m-gateway/test/api/eservices/approveDelegatedEServiceDescriptor.test.ts

```diff
@@ -36,7 +36,6 @@ describe("POST /eservices/:eserviceId/descriptors/:descriptorId/approve router t
         `${appBasePath}/eservices/${eServiceId}/descriptors/${descriptorId}/approve`
       )
       .set("Authorization", `Bearer ${token}`)
-      .set("X-Correlation-Id", generateId())
```

Comment:
Unrelated, but I found out some of these occurrences where we set an unneeded correlationId in M2M tests, so I removed it everywhere